### PR TITLE
[EDU-4133] feat: add Gatsby boilerplate guide.

### PR DESCRIPTION
**Pull request by MarianaAguilera**
### Related issue:

[EDU-4133]

### Changes

- Add Gatsby boilerplate guide PT and EN.
- Add link to Guides pages

### Additional links

n/a.

[EDU-4133]: https://aziontech.atlassian.net/browse/EDU-4133?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/guides/marketplace/templates/gatsby-boilerplate.mdx [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+To enable them:
+
+1. Access [Azion Console](/en/documentation/products/guides/how-to-access-azion-console/) > **Account menu**.
+2. Select the **Billing & Subscriptions** option.
+3. Select the **Subscriptions** tab.
+4. Activate the switch for each module.
+
+:::caution[warning]
+If these modules are activated, the execution of this template could generate usage-related costs. Check the [pricing page](https://www.azion.com/en/pricing/) for more information.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/guides/marketplace/templates/gatsby-boilerplate.mdx:51]: vamos mudar os links para o pricing da doc?

File: src/content/docs/pt-br/pages/guias/marketplace/templates/gatsby-boilerplate.mdx [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+Agora, seu projeto está pronto para trabalhar com um fluxo de trabalho de implantação contínua, atualizando instantaneamente quaisquer alterações no aplicativo ou no repositório.
+
+### Adicione um domínio personalizado
+
+A edge application criada tem um domínio Azion atribuído para torná-la acessível através do navegador. O domínio tem o seguinte formato: `xxxxxxxxxx.map.azionedge.net`. No entanto, você pode adicionar um domínio personalizado para que os usuários acessem sua aplicação por meio dele.
+
+import Button from '~/components/Button.astro';
+
+<Button href="/pt-br/documentacao/produtos/build/edge-application/domains/" text="saiba mais sobre domínios" variant="secondary" />
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/gatsby-boilerplate.mdx:147]: em EN linka para um guia

**Review by hannah-sk — APPROVED** [review]

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/pt-br/pages/guias/marketplace/templates/gatsby-boilerplate.mdx (hunk 2 above)
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/gatsby-boilerplate.mdx:147]:
```suggestion
<Button href="/pt-br/documentacao/produtos/guias/configurar-dominio/" text="saiba mais sobre domínios" variant="secondary" /> 
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/gatsby-boilerplate.mdx [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+    with:
+        ....
+        azionPersonalToken: ${{ secrets.AZION_PERSONAL_TOKEN }}
+        ....
+
+```
+
+3. Abra um pull request com as alterações na main branch e inicie a implantação automática.
+
+Agora, seu projeto está pronto para trabalhar com um fluxo de trabalho de implantação contínua, atualizando instantaneamente quaisquer alterações no aplicativo ou no repositório.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/gatsby-boilerplate.mdx:139]:
```suggestion
Agora, seu projeto está pronto para trabalhar com um fluxo de trabalho de implantação contínua, atualizando instantaneamente quaisquer alterações na aplicação ou no repositório.
```

File: src/content/docs/en/pages/guides/marketplace/templates/gatsby-boilerplate.mdx (hunk 1 above)
Comment (MarianaAguilera) [reply src/content/docs/en/pages/guides/marketplace/templates/gatsby-boilerplate.mdx:51]: deixei uma tarefa para fezer isso logo

File: src/content/docs/en/pages/guides/marketplace/templates/gatsby-boilerplate.mdx [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+While generating your GitHub personal token, grant your scope with the permissions to authorize an OAuth app or a personal token to access public and private repositories, including read and write access to code. You must also enable the **workflow** option to allow adding and updating GitHub Actions workflow files. 
+
+Note that permissions can be scoped either to a user, an organization, or a repository. Read the [Creating a personal access token documentation](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-personal-access-token-classic) for more information.
+:::
+
+You must also activate:
+<Badge>
+[Edge Functions](/en/documentation/products/build/edge-application/edge-functions/)
+</Badge>
+<br/>
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/marketplace/templates/gatsby-boilerplate.mdx:38]:
```suggestion
```

File: src/content/docs/en/pages/guides/marketplace/templates/gatsby-boilerplate.mdx [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+
+Note that permissions can be scoped either to a user, an organization, or a repository. Read the [Creating a personal access token documentation](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-personal-access-token-classic) for more information.
+:::
+
+You must also activate:
+<Badge>
+[Application Accelerator](/en/documentation/products/build/edge-application/application-accelerator/)
+</Badge>
+
+To enable them:
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/marketplace/templates/gatsby-boilerplate.mdx:39]:
```suggestion
To enable it:
```

File: src/content/docs/en/pages/guides/marketplace/templates/gatsby-boilerplate.mdx [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+
+1. Access [Azion Console](/en/documentation/products/guides/how-to-access-azion-console/) > **Account menu**.
+2. Select the **Billing & Subscriptions** option.
+3. Select the **Subscriptions** tab.
+4. Activate the switch for each module.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/marketplace/templates/gatsby-boilerplate.mdx:44]:
```suggestion
4. Activate the switch for the module.
```

File: src/content/docs/en/pages/guides/marketplace/templates/gatsby-boilerplate.mdx [hunk 7]
```diff
@@ -0,0 +1,149 @@
...
+
+:::caution[warning]
+If these modules are activated, the execution of this template could generate usage-related costs. Check the [pricing page](https://www.azion.com/en/pricing/) for more information.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/marketplace/templates/gatsby-boilerplate.mdx:47]:
```suggestion
If a module is activated, the execution of this template could generate usage-related costs. Check the [pricing page](https://www.azion.com/en/pricing/) for more information.
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/gatsby-boilerplate.mdx [hunk 8]
```diff
@@ -0,0 +1,153 @@
...
+Ao gerar seu personal token do GitHub, conceda ao seu scope permissões para autorizar um aplicação OAuth ou um personal token a acessar repositórios públicos e privados, incluindo acesso de leitura e reescrita de código. Você também deve habilitar a opção de workflow para permitir a adição e atualização de arquivos de workflow de GitHub Actions.
+
+Note que as permissões podem ter escopo para um usuário, uma organização ou um repositório. Leia a documentação sobre a [criação de um personal token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-personal-access-token-classic) para obter mais informações.
+:::
+
+Você também deve ativar:
+<Badge>
+[Edge Functions](/pt-br/documentacao/produtos/build/edge-application/edge-functions/)
+</Badge>
+<br/>
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/gatsby-boilerplate.mdx:38]:
```suggestion
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/gatsby-boilerplate.mdx [hunk 9]
```diff
@@ -0,0 +1,153 @@
...
+<Badge>
+[Application Accelerator](/pt-br/documentacao/produtos/build/edge-application/application-accelerator/)
+</Badge>
+
+Para habilitar esses módulos:
+
+1. Acesse o [Azion Console](/pt-br/documentacao/produtos/guias/como-acessar-o-azion-console/) > **Account menu**.
+2. Selecione a opção **Billing & Subscriptions**.
+3. Selecione a aba **Subscriptions**.
+4. Ative o switch para cada módulo.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/gatsby-boilerplate.mdx:48]:
```suggestion
4. Ative o switch do módulo.
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/gatsby-boilerplate.mdx [hunk 10]
```diff
@@ -0,0 +1,153 @@
...
+
+:::caution[atenção]
+Se estes módulos estiverem ativados, a execução deste template pode gerar custos relacionados ao uso. Consulte a [página de preços](https://www.azion.com/pt-br/precos/) para mais informações.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/gatsby-boilerplate.mdx:51]:
```suggestion
Se um módulo estiver ativado, a execução deste template pode gerar custos relacionados ao uso. Consulte a [página de preços](https://www.azion.com/pt-br/precos/) para mais informações.
```

**Pull request merged**